app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Api::BaseController < ApplicationController
   layout nil
+
+  # Method to log login attempts
```

Comment:
Ensure that 'user_id', 'attempted_at', 'successful', and 'ip_address' are properly validated and sanitized to prevent any malicious data from being logged. This is crucial for security, especially since 'ip_address' could be spoofed.